.travis.yml

```diff
@@ -144,6 +166,13 @@ install:
     # Install matplotlib
     pip install -ve .
 
+before_script:
+  - |
+    if [[ $TRAVIS_OS_NAME != 'osx' ]]; then
+      export DISPLAY=:99.0
+      sh -e /etc/init.d/xvfb start
+    fi
```

Comment:
Travis docs [suggest a sleep here](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI).